packages/theme/src/types.ts

```diff
@@ -64,6 +64,8 @@ type PaletteAdditions = {
   banner: {
     info: string;
     error: string;
+    textColor: string;
```

Comment:
The context here is already a palette of colors, so no need to suffix with `Color`, `text` is enough imo